functions/check-started/index.js

```diff
@@ -32,10 +33,14 @@ module.exports = async function (context, checkStartMessage) {
     throw error
   }
 
-  // Delete the row in the preparedCheck table - prevent pupils logging in again.
+  // Delete the row in the preparedCheck table for live checks only - prevent pupils logging in again.
   try {
-    await deleteFromPreparedCheckTableStorage(azureTableService, checkStartMessage.checkCode, context.log)
-    context.log('SUCCESS: pupil check row deleted from preparedCheck table')
+    const check = await sqlFindCheckByCheckCode(checkStartMessage.checkCode, context.log)
+    context.log('check: ', check)

```

Comment:
This check var looks like debug. Perhaps it should be removed?

Reply:
i would agree.  if the object is sizeable then it creates a lot of noise in the logs

Reply:
@jon-shipley @GuyHarwood this is removed.